packages/mui-material/src/Modal/Modal.d.ts

```diff
@@ -88,7 +118,7 @@ export type ModalClassKey = keyof NonNullable<ModalTypeMap['props']['classes']>;
  *
  * - [Modal API](https://mui.com/material-ui/api/modal/)
  */
-declare const Modal: ExtendModalUnstyled<ModalTypeMap>;
+declare const Modal: ExtendModal<ModalTypeMap>;
```

Comment:
I don't think it's necessary to define the `ExtendModal` type. We use such types when we inherit components (for example when Button inherits from ButtonBase). Since nothing inherits from Modal, this type is not needed.

Reply:
@michaldudak Makes sense. Removed it.

Reply:
This also applied to ExtendModalTypeMap. Sorry - I should have been more precise.

Reply:
@michaldudak It's fine. I removed `ExtendModalTypeMap` also. Thanks!